src/metabase/lib/expression.cljc

```diff
@@ -534,6 +535,7 @@
                    (lib.util.match/match-one expr :offset))
           {:message  (i18n/tru "OFFSET is not supported in custom filters")
            :friendly true})
-        (when (= (first expr) :value)
+        (when (and (= (first expr) :value)
```

Comment:
Slipped my noticed on the first pass, but I guess this should use `is-clause?` or similar. Will push an update.